vllm/attention/ops/ipex_attn.py

```diff
@@ -0,0 +1,120 @@
+from typing import Dict, List, Optional, Tuple
+
+import intel_extension_for_pytorch.llm.modules as ipex_modules
+import torch
+
+from vllm import _custom_ops as ops
+
+
+class PagedAttention:
+
+    @staticmethod
+    def get_supported_head_sizes() -> List[int]:
+        return [64, 80, 96, 112, 128, 256]
```

Comment:
Is this correct? Just wondering because the list is identical to the exiting PagedAttention backend.

Reply:
Actually, IPEX has no head size restriction, it will handle the tail.